diff -Nur a/usr/share/applications/viber.desktop b/usr/share/applications/viber.desktop
--- a/usr/share/applications/viber.desktop	2013-12-27 09:58:32.000000000 +0200
+++ b/usr/share/applications/viber.desktop	2015-08-05 15:16:33.857596674 +0300
@@ -1,10 +1,9 @@
 [Desktop Entry]
 Name=Viber
 Comment=Viber VoIP and messenger
-Exec=/opt/viber/Viber
-Icon=/usr/share/pixmaps/viber.png
+Exec=viber
+Icon=viber
 Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Network;Application;
-Path=
+Categories=Network;InstantMessaging;Telephony;
