a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -144,9 +144,9 @@ WINDRES = windres
 	$(WINDRES) $(WINDRESFLAGS) -o $@ $<
 
 install-data-hook:
-	$(MKDIR_P) "${pkgdatadir}"
-	chmod u+w "${pkgdatadir}"
-	cp -rv "${top_srcdir}/data" "${pkgdatadir}"
+	$(MKDIR_P) "${DESTDIR}${pkgdatadir}"
+	chmod u+w "${DESTDIR}${pkgdatadir}"
+	cp -rv "${top_srcdir}/data" "${DESTDIR}${pkgdatadir}"
 
 uninstall-local:
 	find "${pkgdatadir}/data" \( -type f -or -type d \) \
a/doc/Makefile.am b/doc/Makefile.am
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -67,9 +67,9 @@ eps:
 		convert $$f "${builddir}/mininim-figures/"`basename $${f%.png}.eps`; done
 
 install-info-local:
-	$(MKDIR_P) "${infodir}"
-	chmod u+w "${infodir}"
-	cp -rv "${srcdir}/mininim-figures" "${infodir}"
+	$(MKDIR_P) "${DESTDIR}${infodir}"
+	chmod u+w "${DESTDIR}${infodir}"
+	cp -rv "${srcdir}/mininim-figures" "${DESTDIR}${infodir}"
 
 uninstall-local:
 	find "${infodir}/mininim-figures" \( -type f -or -type d \) \
